src/timeline/Timeline.tsx

```diff
@@ -167,25 +173,26 @@ const Timeline = (props: TimelineProps) => {
         screenWidth: width / numberOfDays,
         dayStart: start,
         overlapEventsSpacing: overlapEventsSpacing / numberOfDays,
-        rightEdgeSpacing: rightEdgeSpacing / numberOfDays
+        rightEdgeSpacing: rightEdgeSpacing / numberOfDays,
+        hourBlockHeight
       });
     });
   }, [pageEvents, start, numberOfDays]);
 
   useEffect(() => {
     let initialPosition = 0;
     if (scrollToNow) {
-      initialPosition = calcTimeOffset(HOUR_BLOCK_HEIGHT);
```

Comment:
there is problem with method calcTimeOffset, by changing hour block height won't return correct result for initialPosition.